### PR TITLE
Don't include closure compiler polyfills in secondary binaries.

### PR DESCRIPTION
**Pull request by cramforce**
Small code size wins across the board.

fixes https://github.com/ampproject/amphtml/issues/4933

### CODE REVIEW

File: build-system/tasks/compile.js [hunk 1]
```diff
@@ -198,6 +198,8 @@ function compile(entryModuleFilenames, outputDir,
         // Transpile from ES6 to ES5.
         language_in: 'ECMASCRIPT6',
         language_out: 'ECMASCRIPT5',
+        rewrite_polyfills: !!(
```
Comment (erwinmombay) [inline build-system/tasks/compile.js]: ah, yeah i was trying `no_rewrite_polyfills` lol

**Comment by erwinmombay** [comment]
some good wins. LGTM

File: test/size.txt [hunk 2]
```diff
@@ -1,62 +1,62 @@
...
- 12.87 kB   |     3.58 kB   |     1.58 kB   |   v0/cache-service-worker-0.1.js
-207.07 kB   |    45.93 kB   |    15.98 kB   |   current-min/f.js / current/integration.js
\ No newline at end of file
+      max   |         min   |             gzip   |   file
+      ---   |         ---   |              ---   |   ---
+  70.5 kB   |     5.62 kB   |          2.45 kB   |   alp.js / alp.max.js
+221.74 kB   |     61.3 kB   |   check-types.js
+774.03 kB   |   143.97 kB   |         45.46 kB   |   shadow-v0.js / amp-shadow.js
+    428 B   |       278 B   |       sw-kill.js
+    537 B   |       382 B   |            sw.js
```
Comment (erwinmombay) [inline test/size.txt]: do you mind rerunning? looks like sw an sw-kill had a problem?

Comment (cramforce) [inline test/size.txt]: Are you saying they are too small?

Comment (cramforce) [inline test/size.txt]: I think this is correct. They do basically nothing.

Comment (erwinmombay) [inline test/size.txt]: oh i mean the filename is in the wrong column. this happens if it cant find the min/max counterpart.

**Comment by cramforce** [comment]
Could you check what happened? The binaries seemed fine and I rebased and reran.

**Comment by erwinmombay** [comment]
@cramforce will do

**Pull request merged**